lang/en.json

```diff
@@ -10,7 +10,7 @@
         "RedisTimeoutFieldTitle": "Redis timeout",
         "RedisTimeoutFieldHelp": "Redis connection timeout in seconds. “0.0” meaning unlimited. Can be a float eg “2.5” for a connection timeout of 2.5 seconds.",
         "NumberOfQueueWorkersFieldTitle": "Number of queue workers",
-        "NumberOfQueueWorkersFieldHelp": "Number of allowed maximum queue workers. Accepts a number between 1 and 16. Best practice is to set the number of CPUs you want to make available for queue processing. Be aware you need to make sure to start the workers manually. We recommend to not use 9-15 workers, rather use 8 or 16 as the queue might not be distributed evenly into different queues.",
+        "NumberOfQueueWorkersFieldHelp": "Number of allowed maximum queue workers. Accepts a number between 1 and 4096. Best practice is to set the number of CPUs you want to make available for queue processing. Be aware you need to make sure to start the workers manually. We recommend to not use 9-15 workers, rather use 8 or 16 as the queue might not be distributed evenly into different queues.",
```

Comment:
@haristku Nothing wrong here, but the way Translations work in Matomo, if you want to change an existing key, you need to create a new one, otherwise this won't get translated.
```suggestion
        "NumberOfQueueWorkersFieldHelp": "Number of allowed maximum queue workers. Accepts a number between 1 and 16. Best practice is to set the number of CPUs you want to make available for queue processing. Be aware you need to make sure to start the workers manually. We recommend to not use 9-15 workers, rather use 8 or 16 as the queue might not be distributed evenly into different queues."
        "NumberOfQueueWorkersFieldHelpNew": "Number of allowed maximum queue workers. Accepts a number between 1 and 4096. Best practice is to set the number of CPUs you want to make available for queue processing. Be aware you need to make sure to start the workers manually. We recommend to not use 9-15 workers, rather use 8 or 16 as the queue might not be distributed evenly into different queues.",
```

You would also need to update the [SystemSettings.php](https://github.com/matomo-org/plugin-QueuedTracking/blob/d2df4b277f5aa5080c53b15258a23c6374088840/SystemSettings.php#L197) to include this new key.

Reply:
ok... i'll update the [SystemSettings.php](https://github.com/matomo-org/plugin-QueuedTracking/blob/d2df4b277f5aa5080c53b15258a23c6374088840/SystemSettings.php#L197) too